src/iop/colorchecker.c

```diff
@@ -121,7 +121,7 @@ const char *aliases()
 
 const char *description(struct dt_iop_module_t *self)
 {
-  return dt_iop_set_description(self, _("perform color space corrections and apply looks."),
+  return dt_iop_set_description(self, _("perform color space corrections and apply looks"),
```

Comment:
unwanted, we have dot on description on all others... maybe a wrong merge conflict resolution ?

Reply:
yes, it's a wrong rebase